src/couch/src/couch_hash.erl

```diff
@@ -31,15 +31,27 @@ md5_hash_update(Context, Data) ->
 -else.
 
 md5_hash(Data) ->
-    crypto:hash(md5, Data).
+    case config:is_enabled(fips) of
+        true -> erlang:md5(Data);
+        false -> crypto:hash(md5, Data)
```

Comment:
I found https://www.erlang.org/doc/apps/crypto/fips.html#avoid-md5-for-hashing which says:
> For exceptional, non-cryptographic use cases one may consider switching to `erlang:md5/1` as well.

Would it be worth linking to that documentation, and/or maybe documenting the difference between these two md5 algorithms?

Reply:
Good idea. So it's clear what this is about.